docs/developer/stock-allocation.mdx

```diff
@@ -128,6 +128,37 @@ the [`updateWarehouse`](developer/api-reference/mutations/update-warehouse.mdx)
 There is an option to change the following warehouse fields: `name`, `slug`, `email`, `address` and
 also define privacy and `click and collect` options.
 
+The warehouse could be defined as private or public. When the warehouse is public, the stock
+is shown, otherwise it isn't.
```

Comment:
```suggestion
The warehouse could be defined as private or public. When the warehouse is public, the stock
information can be queried publicly in API. Otherwise, it requires permission.
```